app/models/concerns/umrdr/generic_work_metadata.rb

```diff
@@ -4,8 +4,14 @@ module GenericWorkMetadata
     extend ActiveSupport::Concern
     included do
       property :subject, predicate: ::RDF::Vocab::MODS.subject
-      property :date_coverage, predicate: ::RDF::Vocab::DC.coverage do |index|
-        index.as :stored_searchable, :facetable
+      property :date_coverage_from, predicate: ::RDF::Vocab::DC.temporal, multiple: false do |index| 
+       index.type :text 
+       index.as :stored_searchable, :facetable
+      end
+
+      property :date_coverage_to, predicate: ::RDF::Vocab::DC.coverage, multiple: false do |index| 
```

Comment:
The coverage predicate is a range.  Here it is being overloaded to mean, "the end of a coverage".